docs/source/guide/ml.md

```diff
@@ -124,7 +124,7 @@ Depending on score types you can select a sampling strategy:
 * prediction-score-min (min is the best score) 
 * prediction-score-max (max is the best score)
  
-See more about active learning sampling in [Set up task sampling for your project](guide/start.html#Set-up-task-sampling-for-your-project). 
+See more about active learning sampling in [Set up task sampling for your project](start.html#Set-up-task-sampling-for-your-project). 
```

Comment:
It's better to put the link here to where we describe prediction score-based filtering/ordering interactions with DM

Reply:
Yeah, that still needs to be written but updated the link to the labeling doc.

Reply:
I'll resolve this more fully with the bigger updates I'll make to this page